packages/integration-tests/index.js

```diff
@@ -123,7 +125,7 @@ async function startWorkflowExecution(workflowArn, inputFile) {
 async function executeWorkflow(stackName, bucketName, workflowName, inputFile) {
   const workflowArn = await getWorkflowArn(stackName, bucketName, workflowName);
   const execution = await startWorkflowExecution(workflowArn, inputFile);
-  const executionArn = execution.executionArn;
+  const { executionArn } = execution;
```

Comment:
These two lines could just be

`const { executionArn } = await startWorkflowExecution(workflowArn, inputFile);`